### ISSUE TITLE
Doesn't seem to do anything

### ISSUE THREAD

**Issue by zaps** [concon121/atom-maven#61]
Loading a minimal project generated with `mvn archetype:generate`, the bouncy "configuring classpath" message appears but never goes away and the package doesn't respond to any changes made to the pom.

![atom-maven](https://cloud.githubusercontent.com/assets/2954949/21058455/06ba2f26-be0c-11e6-834e-671e866d33c5.gif)

I don't have Maven in my path so I'm configuring it via `mavenHome`.  Neither  "`/home/user/tools`" nor "`~/tools`" conventions seem to work.

I must be missing something simple here :stuck_out_tongue_winking_eye:

**Comment by concon121** [comment]
I'll take a closer look this weekend, but in the meantime if you get a change can you see what gets logged out into the console (ctrl + shift + i)?  

EDIT:
Its also worth noting that .gitignore'd files wont get found.

**Comment by zaps** [comment]
Console appears to be empty during any operations on a pom. Tried on both Mac and Linux.

**Comment by concon121** [comment]
I (finally) released the new version earlier today.  I've removed the need for configuration, so as long as maven is correctly installed on your computer it should work.

I made a small adjustment to the message panel for the sake of UX.  It should only show the "atom-maven is configuring your classpath" message if it finds a pom file in your workspace.  If you dont see this message, it means your pom file isn't visible to ```atom.workspace.scan```.

Let me know how it goes!
